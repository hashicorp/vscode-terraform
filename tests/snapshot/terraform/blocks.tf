resource "aws_instance" "web" {
  ami           = "ami-a1b2c3d4"
  instance_type = "t2.micro"
  timeouts {
    create = "60m"
    delete = "2h"
  }
}
