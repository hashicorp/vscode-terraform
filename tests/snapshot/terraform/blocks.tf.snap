>resource "aws_instance" "web" {
#^^^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#        ^ source.terraform meta.block.terraform
#         ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#          ^^^^^^^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                      ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                       ^ source.terraform meta.block.terraform
#                        ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#                         ^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                            ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                             ^ source.terraform meta.block.terraform
#                              ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  ami           = "ami-a1b2c3d4"
#^^ source.terraform meta.block.terraform
#  ^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#     ^^^^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform
#                ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#                 ^ source.terraform meta.block.terraform variable.declaration.terraform
#                  ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                   ^^^^^^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                               ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>  instance_type = "t2.micro"
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#               ^ source.terraform meta.block.terraform variable.declaration.terraform
#                ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#                 ^ source.terraform meta.block.terraform variable.declaration.terraform
#                  ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                   ^^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                           ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>  timeouts {
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^ source.terraform meta.block.terraform meta.block.terraform entity.name.label.terraform
#          ^ source.terraform meta.block.terraform meta.block.terraform
#           ^ source.terraform meta.block.terraform meta.block.terraform punctuation.section.block.begin.terraform
>    create = "60m"
#^^^^ source.terraform meta.block.terraform meta.block.terraform
#    ^^^^^^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#          ^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform
#           ^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#            ^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform
#             ^ source.terraform meta.block.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#              ^^^ source.terraform meta.block.terraform meta.block.terraform string.quoted.double.terraform
#                 ^ source.terraform meta.block.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>    delete = "2h"
#^^^^ source.terraform meta.block.terraform meta.block.terraform
#    ^^^^^^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#          ^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform
#           ^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#            ^ source.terraform meta.block.terraform meta.block.terraform variable.declaration.terraform
#             ^ source.terraform meta.block.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#              ^^ source.terraform meta.block.terraform meta.block.terraform string.quoted.double.terraform
#                ^ source.terraform meta.block.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>  }
#^^ source.terraform meta.block.terraform meta.block.terraform
#  ^ source.terraform meta.block.terraform meta.block.terraform punctuation.section.block.end.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>